README.md

```diff
@@ -67,3 +67,18 @@ A typical Prism response includes the following top level fields:
 Using prism from the command line
 ------------------------------
 There is a [ruby gem](./marauder/README.md) that allows you to query the api from the command line.
+
+
+Development
+------------------------------
+1. Get Security Janus credentials.
+
+2. Ensure you are running Java 11
+
+3. Update the `play.http.secret.key` configuration in `conf/application.conf` to a dummy value
```

Comment:
Additionally: create the gc log file if it doesn't already exist?

Reply:
Also, thinking about it, why doesn't the `application.conf` have a dummy value already?

Reply:
Turns out these are not required if you start the app using `run` instead of `start`!